src/cvxpy_gurobi/translation.py

```diff
@@ -322,6 +322,9 @@ def visit_AddExpression(self, node: AddExpression) -> Any:
     def visit_Constant(self, const: cp.Constant) -> Any:
         return const.value
 
+    def visit_Parameter(self, param: cp.Parameter) -> Any:
+        return param.value
```

Comment:
We'll want to raise an error if the parameter's value wasn't set yet. You can do it in this PR, or I can do it in a follow-up, it's up to you.

Reply:
Thanks for the feedback @jonathanberthias ! Added an exception and a test for both happy/non-happy paths in [this](https://github.com/jonathanberthias/cvxpy-gurobi/pull/67/commits/1b832e6a96b06ca162889ed0dc4773a797092c70) commit.